src/fprime_gds/executables/cli.py

```diff
@@ -782,3 +782,41 @@ def get_arguments(self) -> Dict[Tuple[str, ...], Dict[str, Any]]:
 
     def handle_arguments(self, args, **kwargs):
         return args
+    
+class OpenMCTTelemetryPollerParser(ParserBase):
```

Comment:
We may want to include the value of the fprime-gds `--tts-port` argument, since if a different port is used when launching fprime-gds, we get an error in fprime-openmct (because it defaults to 50050).

Reply:
Fixed! I was not parsing the StandardPipeline arguments before and added that in the updated commit